charon/pkgs/maven.py

```diff
@@ -383,6 +388,30 @@ def handle_maven_uploading(
                 failed_metas.extend(_failed_metas)
                 logger.info("archetype-catalog.xml updating done in bucket %s\n", bucket_name)
 
+        # 10. Generate signature file if gpg ket is provided
+        if (key_id is not None or key_file is not None) and passphrase is not None:
+            artifacts = [s for s in valid_mvn_paths if s.endswith(".jar") or s.endswith(".pom")]
```

Comment:
Doe this mean we only cares about ".jar" and ".pom" file? If so I don't think it is reasonable. In maven, many files are valid artifacts. I think we should use a configuration to mark the file types which are not needed to do signing, like ".asc" or similar.